### PR TITLE
Fix cuda graph mode in flashinfer attn backend

### PR DESCRIPTION
**Pull request by benbarsdell**
cc @kushanam

## Motivation

Cuda graph mode causes an error when running Llama models on Blackwell systems.

## Modifications

Changes the scale factors passed to the backend kernel to use the float versions instead of the Parameter versions. This ensures they don't end up triggering device-to-host copies that synchronize the device and break cuda graph capture.

## Accuracy Tests

N/A

## Benchmarking and Profiling

N/A

## Checklist

- [ ] Format your code according to the [Format code with pre-commit](https://docs.sglang.ai/developer_guide/contribution_guide.html#format-code-with-pre-commit).
- [ ] Add unit tests according to the [Run and add unit tests](https://docs.sglang.ai/developer_guide/contribution_guide.html#run-and-add-unit-tests).
- [ ] Update documentation according to [Write documentations](https://docs.sglang.ai/developer_guide/contribution_guide.html#write-documentations).
- [ ] Provide accuracy and speed benchmark results according to [Test the accuracy](https://docs.sglang.ai/developer_guide/contribution_guide.html#test-the-accuracy) and [Benchmark the speed](https://docs.sglang.ai/developer_guide/contribution_guide.html#benchmark-the-speed).

### CODE REVIEW

File: python/sglang/srt/layers/attention/flashinfer_backend.py [hunk 1]
```diff
@@ -501,8 +501,9 @@ def forward_extend(
                 sm_scale=layer.scaling,
                 window_left=layer.sliding_window_size,
                 logits_soft_cap=logits_soft_cap,
-                k_scale=layer.k_scale,
-                v_scale=layer.v_scale,
+                # Must use _float to avoid device-to-host copy that breaks cuda graph capture.
```
Comment (zhyncs) [inline python/sglang/srt/layers/attention/flashinfer_backend.py:504]: can you add a unit test

**Review by hnyls2002 — APPROVED** [review]
